Packs/FeedMitreAttackv2/ReleaseNotes/1_1_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MITRE ATT&CK
+
+Fixed an issue in the **TAXII2ApiModule** script.
```

Comment:
Is this using the TAXII2ApiModule? Does it use the authentication headers?

Reply:
no